Common/cpp/ReanimatedRuntime/WorkletRuntime.cpp

```diff
@@ -4,15 +4,64 @@
 #include "WorkletRuntimeCollector.h"
 #include "WorkletRuntimeDecorator.h"
 
+#include <jsi/decorator.h>
+
 namespace reanimated {
 
+struct AroundLock {
+  std::recursive_mutex *mutex;
```

Comment:
It looks weird as a raw pointer but I guess that's fine.
```suggestion
  std::recursive_mutex *mutex_;
```

Reply:
We receive pointer to runtime mutex in `LockableRuntime` constructor, but he `std::recursive_mutex` class doesn't have a copy assignment operator or copy constructor defined, so thats why here is raw pointer.